check/configuration/configuring-existing-linters/README.md

```diff
@@ -1,248 +1,24 @@
-
-### Pylint
-
-You may specify additional pylint plugins in your `.pylintrc`, using the line `load-plugins=...`
-
-if you want to run the plugin `pylint-django` as part of your setup, you would add the line
-
-`load-plugins=pylint_django` to your `.pylintrc`, but you **also** need to tell trunk to install the package:
-
-```yaml
-- pylint@2.11.0:
-    packages:
-      - pylint-django@2.4.4
-      ...
-```
-
-### Markdownlint
-
-Older versions of markdownlint had a bug where it printed plaintext output even when run with `--json`. We rely on JSON output so we can parse and ingest the results from markdownlint. The package we use for markdownlint is actually [markdownlint-cli](https://www.npmjs.com/package/markdownlint-cli); `>= 0.29.0` is verified to work.
-
-### Rustfmt
-
-We currently use the version of `rustfmt` packaged with rust, so for `rustfmt` version, specify your rust version (for example `rustfmt@1.61.0`).
-
-If you have `edition` in your `cargo.toml`, `rustfmt` also needs the same information in `.rustfmt.toml` in your repo root. For example, your `.rustfmt.toml` might contain:
-
-```toml
-edition = "2021"
-```
-
-### Sqlfluff
-
-Sqlfluff is only configured as a linter by default because its formatting capabilities are limited. To turn sqlfluff formatting on, enable its subcommand:
-
-```yaml
-lint:
-  enabled:
-    - sqlfluff@<version>:
-        commands: [lint, fix]
-```
-
-### Terraform
-
-We currently support `terraform validate` and `terraform fmt`, but only `fmt` is enabled by default when you add `terraform` to your enabled list in `trunk.yaml`. To enable `validate`, add this to your `trunk.yaml`:
-
-```yaml
-lint:
-  enabled:
-    - terraform@<version>:
-        commands: [validate, fmt]
-```
-
-Note: you must run `terraform init` before running `trunk check` with `terraform validate` enabled (both locally, or on CI).
+# Supported Linters
+
+List of linters supported out of the box by Trunk Check
+
+Enable any of the following tools with:
+
+trunk check enable linter
+
+| Technology | Linters |
+|------------|---------|
+| C++ | [clang-tidy](./clang-tidy.md)|
+| infra | [golangci-lint](./golangci-lint.md), [Terraform](./terraform.md), [SQLFluff](./sqlfluff.md), [Ansible-lint](./ansible-lint.md), [Actionlint](./actionlint.md)|
+| rust | [rustfmt](./rustfmt.md), [Clippy](./clippy.md)|
+| JavaScript | [ESLint](./eslint.md), [Prettier](./prettier.md)|
+| Java | [Detekt](./detekt.md)|
+| C | [ClangFormat](./clang-format.md)|
+| yaml | [Yamllint](./yamllint.md)|
+| Security | [Trufflehog](./trufflehog.md), [Checkov](./checkov.md), [Brakeman](./brakeman.md), [Trivy](./trivy.md), [Gitleaks ](./gitleaks.md), [OSV-Scanner](./osv-scanner.md)|
+| nodejs | [SVGO](./svgo.md)|
+| shell | [ShellCheck](./shellcheck.md)|
+| Python | [Ruff](./ruff.md), [Black](./black.md), [Bandit](./bandit.md), [Isort](./isort.md), [Pylint](./pylint.md), [Flake8](./flake8.md)|
+| assets | [Oxipng](./oxipng.md)|
+| markdown | [Markdownlint](./markdownlint.md)|
```

Comment:
I'm confused. Where is this list coming from? It's missing a log and also these are not accurate (e.g. detekt is kotlin not java, clang-format is also c++...)

Reply:
These are coming from the 'group' setting in Dato. They were originally for grouping the marketing linter pages.  We could have an alternative set of groups, which is what I'm doing for the linters that aren't in Dato yet.